src/applications/static-pages/events/components/Events/styles.scss

```diff
@@ -0,0 +1,19 @@
+.events {
```

Comment:
Can this Sass go in the global /site-wide area? we're clearing all the Sass out of static pages

Reply:
Oh interesting, how come?

Reply:
it's a step we're taking to help isolating the apps. This one is a mess right now, so we're trying to limit what's in there until it gets rearchitected

Reply:
Updated + pushed btw, but lmk if you want me to change this in other apps too (e.g. header v2 I think also has its own styling files)

Reply:
The idea is to just put all custom styling in `shame.scss`? We should probably rename that file to `globalStyles.scss` or something since sometimes we just need custom styling to achieve a certain design we've been handed 🙂